trin-cli/src/main.rs

```diff
@@ -54,7 +56,7 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
 
 fn build_request<'a>(
     method: &'a str,
-    raw_params: &'a Option<[Box<RawValue>; 1]>,
+    raw_params: &'a Option<Vec<Box<RawValue>>>,
```

Comment:
I'm not sure what functionality we have for this CLI, or how extensively it was tested initially, but did we test both `Ping` and `FindContent` after this change?

Reply:
Yeah, tests would require serving some kind of static content from a database. We're sort of in a limbo state between two databases models right now - so it felt out of scope to address that in this pr. This was just tested locally, I snuck some code in that manually inserted random values into the db and then sent it over the wire to another client (and I also tested `Ping` to make sure that didn't break). Not ideal, but enough to get some confidence that the basic functionality is there. Also, imo tests for `FindContent`/`Content` are so glaringly obvious that I'm not worried about them falling through the cracks as we get the db sorted / peertest infrastructure up.